code/create_actor.py

```diff
@@ -0,0 +1,33 @@
+import sqlite3
+
+# Connect to the SQLite database
+conn = sqlite3.connect('data/rotten_tomatoes.db')
+cursor = conn.cursor()
+
+# Truncate (clear) the movie_actors table if it already exists
+cursor.execute("DROP TABLE IF EXISTS movie_actors;")
+
+# Create the movie_actors table with actor_id set to NULL for all entries
+cursor.execute("""
+    CREATE TABLE movie_actors (
+        actor_id INTEGER NULL,
+        movie_id INTEGER,
+        actor_name TEXT,
+        FOREIGN KEY (movie_id) REFERENCES rotten_tomatoes_movies_dicaprio_winslet(movie_id)
+    );
```

Comment:
Reyndar er `NULL` default, þannig það væri betra að skýra hvaða dálkar eru EKKI null. 
```suggestion
CREATE TABLE movie_actors (
    actor_id INTEGER 
    movie_id INTEGER NOT NULL,
    actor_name TEXT NOT NULL,
    FOREIGN KEY (movie_id) REFERENCES rotten_tomatoes_movies_dicaprio_winslet(movie_id)
);
```